app/templates/home.html

```diff
@@ -1,38 +0,0 @@
-{% extends "base.html" %}
```

Comment:
Is this deleted on purpose?

Reply:
I removed This file because it is no longer in use.